OpenSim/Common/Reporter.h

```diff
@@ -113,6 +113,19 @@ class Reporter : public AbstractReporter {
     OpenSim_DECLARE_LIST_INPUT(inputs, InputT, SimTK::Stage::Report,
         "Variable list of quantities to be reported.");
 
+    //=============================================================================
+    // PUBLIC METHODS
+    //=============================================================================
+
+    // Allow overloading of updInput
+    using AbstractReporter::updInput;
```

Comment:
This line essentially pulls the `updInput` with one argument into scope. Without this line:
`'OpenSim::Reporter<InputT>::updInput': function does not take 1 arguments`. This also occurs if you put the line inside `updInput()` or use `this->updInput("inputs")`.
http://stackoverflow.com/questions/1734893/overloading-a-method-in-a-subclass-in-c
http://stackoverflow.com/questions/411103/function-with-same-name-but-different-signature-in-derived-class

Although this does what we want, it isn't the cleanest. I'm open to suggestions that seem less like a workaround.